docs/using-concourse/fly-cli.any

```diff
@@ -78,6 +78,27 @@ flag. Once you've learned what the commands do, you may want to consult
   \code{example} Concourse instance.
 }
 
+\section{\code{login}\aux{: Authenticating with and saving Concourse targets}}{fly-login}{
+
+  There are cases when you would like to remove all evidence of a particular
+  configuration. This is achieve by the \code{logout} command. There are two
```

Comment:
should probably say 'target' - 'configuration' is a bit vague

also achieve -> achieved